deploy/lib/createContainers.js

```diff
@@ -24,6 +24,84 @@ module.exports = {
     });
   },
 
+  applyDomainsContainer(containerId, customDomains) {
+    // we make a diff to know which domains to add or delete
+    const domainsToCreate = [];
+    const domainsIdToDelete = [];
+    const existingDomains = [];
+
+    this.listDomainsContainer(containerId).then((domains) => {
+      domains.forEach((domain) => {
+        existingDomains.push({ hostname: domain.hostname, id: domain.id });
+      });
+
+      if (
+        customDomains !== undefined &&
+        customDomains !== null &&
+        customDomains.length > 0
+      ) {
+        customDomains.forEach((customDomain) => {
+          domainsIdToDelete.push(customDomain.id);
+
+          let domainFound = false;
+
+          existingDomains.forEach((existingDom) => {
```

Comment:
I think there is a better way to do this, for loop in a for loop is not a good practice